routes/api/login.js

```diff
@@ -15,18 +16,19 @@ router.post("/login", async function postLogin(req, res, next) {
     const {
       uid, name, picture: profileUrl,
     } = await firebase.auth().verifyIdToken(idToken);
-    const userData = await User.findOrCreate({
+
+    const { doc: user } = await User.findOrCreate({
       uid,
-    }).update({
+    });
+
+    const result = user.update({
```

Comment:
result가 사용되는 곳이 안 보이네요! if문을 넣어서 user.profileUrl과 user.name이 같지 않을 때만 업데이트 해줄 수 있을 것 같습니다.
```suggestion
    if (user.profileUrl !== profileUrl || user.name !== name) {
      await user.update({
    }
```